andela/wg-bigas-opus
# Improve test coverage

**Description of Task to be completed?**
* Improve the test coverage by testing further.

**How should this be manually tested?**
* Open wg-bigas-opus repository on Github and check the travis bagde for the test coverage.
* Click on the build and it will redirect you to travis where you can see the coverage.

**Screenshots**
**Build badge**
![image](https://user-images.githubusercontent.com/30326444/40496016-683fb78a-5f81-11e8-862e-256de78e30a0.png)

**Coverage**
![image](https://user-images.githubusercontent.com/30326444/40496029-6f830c72-5f81-11e8-995f-509bd8bb70bb.png)
